docs/policygenerator-reference.yaml

```
@@ -60,6 +60,15 @@ policyDefaults:
   # annotation. This defaults to ["NIST SP 800-53"].
   standards:
     - "NIST SP 800-53"
+  # Optional. Array of policysets that policy is going to be part of. policyset details can be defined
+  # in policysets section. When a policy is part of a policyset, placementbinding will no longer be 
+  # generated for policy. Set policies[*].generatePolicyPlacement or policyDefaults.generatePolicyPlacement
+  # to override.
+  policysets: []
```

A camel cased field here would match the other fields. (Also need to update the `generatePolicyPlacement` field with the name that's decided upon.)
```suggestion
  # Optional. Array of policy sets that the policy will join. Policy set details can be defined
  # in the policySets section. When a policy is part of a policy set, a placement binding will not be 
  # generated for the policy since one is generated for the set. Set policies[*].generatePolicyPlacement or policyDefaults.generatePolicyPlacement
  # to override.
  policySets: []
```